### DISCUSSION TITLE
Show tests which covers a line

### DISCUSSION THREAD

**Discussion by christianbumann** [OpenCover/opencover#1005] [General]
It would be nice to have the information which test(s) covers a specific line, not only the information if the line is covered or not.
Is this currenty supported? If yes is a flag required? Which test system is supported (NUnit, MSTest)?

**Comment by sawilde** [comment]
you can use the -coverbytest switch but be warned it is slow due to the additional instrumentation

**Reply by christianbumann** [reply]
Thank you for the answer
